composition-js/src/compose.ts

```diff
@@ -30,7 +31,15 @@ export interface CompositionSuccess {
   errors?: undefined;
 }
 
-export function compose(subgraphs: Subgraphs): CompositionResult {
+export interface ComposeOptions {
```

Comment:
I would prefer to call it `CompositionOptions` but this name is already exported from `merge` :\

I decided not to rename the existing interface to use it for the new thing to avoid breaking somebody's code in a confusing way: same name, totally different usage and meaning.